app/uk/gov/hmrc/agentepayeregistration/validators/AgentEpayeRegistrationValidator.scala

```diff
@@ -113,4 +128,26 @@ object AgentEpayeRegistrationValidator {
       Valid(())
     else
       Invalid(Failure("INVALID_FIELD", s"The $propertyName field is not a valid phone number"))
+
+  private[validators] def isInPast(date: LocalDate)(paramName: String) = {
+    val today = LocalDate.now(DateTimeZone.UTC)
+    if (today.isAfter(date))
+      Valid(())
+    else
+      Invalid(Failure("INVALID_DATE_RANGE", s"'$paramName' date must be in the past"))
+  }
+
+  private[validators] def isValidDateRange(from: LocalDate, to: LocalDate) = {
+    Try(from.isAfter(to)) match {
```

Comment:
I think we could simplify this again assuming date is never null, no exception is thrown? wdyt?

Reply:
Sure, I'll change it back. It was previously making that assumption, but I changed it after another PR comment mentioned `isAfter` could throw an exception. The only cases where it'll throw an exception are when the parameter is null though.